aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -115,5 +115,28 @@ Tensor& set_quantizer_(Tensor& self, ConstQuantizerPtr quantizer) {
   return self;
 }
 
+Tensor quantized_clone(const Tensor& self) {
+  TORCH_CHECK(
+      isQIntType(self.scalar_type()),
+      "Input tensor must have a quantized ScalarType");
+  auto quantizer = get_qtensorimpl(self)->quantizer();
+
+  TORCH_CHECK(quantizer->qscheme() == kPerTensorAffine);
+  auto scale = static_cast<PerTensorAffineQuantizer*>(quantizer.get())->scale();
+  auto zero_point =
```

Comment:
`self.q_zero_point()`

please take a look at the wiki that introduces quantized tensor API